develop/schemadoc/Protocol/Protocol.Params.Param.Type-options.md

```diff
@@ -194,22 +194,48 @@ For more information, refer to [Multi-threaded timers SNMP](xref:AdvancedMultiTh
 
 ### ssh pwd
 
-This option specifies that this parameter holds the password used for setting up SSH communication based on user credentials. When using this option, there should also be another parameter that uses the "ssh username" option. The SSH connection is established by DataMiner itself, if the parameters are filled in correctly.
+> [!IMPORTANT]
+> Link this parameter via the [Password](xref:Protocol.PortSettings.SSH.Credentials.Password) element 
+> instead of using this **ssh pwd** option.
+> This option only supports a single ssh connection and only supports port 22.
+
+This option specifies that this parameter holds the password 
+for setting up SSH communication based on user credentials. 
+When using this option, there should also be another parameter that uses the "ssh username" option. 
+The SSH connection is established by DataMiner itself, if the parameters are filled in correctly.
 
 Only applicable for parameters of type read.
 
+> [!WARNING]
+> Use the [password](xref:Protocol.Params.Param.Measurement.Type-options#options-for-measurement-type-string) option
+> when this parameter is displayed.
+> Using this option ensures greater security.
+
 ### ssh username
 
-This option specifies that this parameter holds the username used for setting up SSH communication based on user credentials. When using this option, there should also be another parameter that uses the "ssh pwd" option. The SSH connection is established by DataMiner itself, if the parameters are filled in correctly.
+> [!IMPORTANT]
+> Link this parameter via the [Username](xref:Protocol.PortSettings.SSH.Credentials.Username) element 
+> instead of using this **ssh username** option.
+> This option only supports a single ssh connection and only supports port 22.
+
+This option specifies that this parameter holds the username 
+for setting up SSH communication based on user credentials. 
+When using this option, there should also be another parameter that uses the "ssh pwd" option. 
+The SSH connection is established by DataMiner itself, if the parameters are filled in correctly.
 
 Only applicable for parameters of type read.
 
 ### ssh options
 
-This option specifies that this parameter holds the required data for setting up SSH communication based on an identity file. This is an alternative way to set up an SSH connection (instead of user credentials). The content of the "ssh options" parameter is as follows:
+> [!IMPORTANT]
+> Link this parameter via the [Identity](xref:Protocol.PortSettings.SSH.Identity) element 
```

Comment:
The usage of this option is not recommended. It only support connectors with a single SSH connection using the fixed port 22. Instead, link this via the [Identity](xref:Protocol.PortSettings.SSH.Identity) element.